test/mavsdk_tests/mavsdk_test_runner.py

```diff
@@ -402,14 +402,23 @@ def start_runners(self,
 
         if self.config['mode'] == 'sitl':
             if self.config['simulator'] == 'gazebo':
+                # first check that vehicle is standard_vtol and only then
+                # if wind change case, as other vehicles don't know about this case
+                if test['vehicle'] == 'standard_vtol'\
```

Comment:
This doesn't really work. Only works when a specific test is loaded seperately (eg `test/mavsdk_tests/mavsdk_test_runner.py test/mavsdk_tests/configs/sitl.json --speed-factor 4 --model standard_vtol --case 'Fly VTOL mission with wind change'`)
 @julianoes would you know a way to load different worlds depending on the selected test case?

Reply:
Really nice work in this PR by the way! Airspeed and wind is definitely harder than I'd imagine.

It's almost like we should encode the world in the test case name. What about something like this:

Call the test case: `Fly VTOL mission with wind change (ramped_up_wind.world)`, and then extract the world like this:

```
import re
# Use regular expression to extract worldname.world from case name

match = re.search(r'\((.*?\.world)\)', case)

if match:
    result = match.group(1)
    world_name = match.group(1)
else:
    world_name = 'empty.world'
```

Reply:
That seems to do the trick! Thanks a lot @julianoes !